keps/sig-node/2133-kubelet-credential-providers/README.md

```diff
@@ -312,6 +313,12 @@ The plugin can signal to the kubelet how it should cache a given response. There
 2. Registry: the kubelet should cache and use this response only for future images with the same registry hostname (and port if included).
 3. Image: the kubelet should cache and use this response only for future images that match the image exactly.
 
+### Metrics
+
+Two kubelet metrics will be added:
+* `kubelet_credential_provider_plugin_errors`: this will track the number errors that occurred from invoking an exec plugin
+* `kubelet_credential_provider_plugin_duration`: this will track the duration of execution by plugins.
+
 ### Test Plan
 
 Alpha:
```

Comment:
Enhancements team [requested you add test plans for beta](https://github.com/kubernetes/enhancements/issues/2133#issuecomment-772850675), can you please add those here?

Reply:
sure thing, thanks for catching that

Reply:
done, thanks